src/assets/tailwind.output.css

```diff
@@ -3,6 +3,12 @@
 @import 'tailwindcss/components';
 @import 'tailwindcss/utilities';
 
+@layer base {
+  html {
+    font-family: 'Nunito', sans-serif;
+  }
+}
+
 /* Custom Components */
```

Comment:
@egiegarciano14 hello Egie, one favor here please. Please ko move sa font-face above the `@layer base`. Sorry I put it previously below sa custom component na comment. 🙇🏼